test.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
1. What this file do?
2. Why it's placed in the root folder of the library?

We already can run `php test/test.php` to run the test suite.